guide/designing-products/open-design.md

```diff
@@ -51,68 +52,105 @@ Illustration sources
+- Open-mindedness
+- Respectful feedback
+- Time management
+- Teamwork
+- Openness & transparency
 
-Designers, developers, translators, project managers, and others work together in the open. Multidisciplinary collaboration, openness, and diversity of people involved make sure that the best solution wins no matter where it comes from.
+## Bitcoin has a spirit of openness
 
-##  Open design and Bitcoin
+We could argue Bitcoin needs open design to stay true to its mission.
 
-The spirit of openness is deeply embedded into Bitcoin; the source code is public, participation in the network is public and unrestricted, and transferring value via the Bitcoin protocol is permissionless. Thousands of projects have borrowed ideas, techniques, and code from Bitcoin to create their open-source experiments. For designers who aim to work collaboratively, it’s a robust foundation.
+* Bitcoin is public and open source
+* It has public & unrestricted network participation
+* It allows permissionless transfer of value in the Bitcoin protocol
+
+Thousands of projects have borrowed ideas, techniques, and code from Bitcoin. Countless open-source experiments have arisen. To a designer, Bitcoin is a robust foundation for collaboration. A foundation that comes with a responsibility toward the benefit of society.
+
+This ethical responsibility encompasses application design. Most people interact with applications instead of the underlying Bitcoin technical protocol. If someone controls access to an application, it won't matter if the protocol is open. It is up to the global community to build principled Bitcoin applications.
+
+Bitcoin is a protocol created to give access to anyone. Access and accessibility go hand in hand. Designing Bitcoin products means working on removing barriers of entry. It means making decisions to create a unique balance of features for ease of use. You should design for a specific audience at the same time as giving the widest access.
 
-We could even argue that Bitcoin needs open design to stay true to its mission, principles, and philosophy. Most users don’t interact directly with technical protocols but with the applications built on top of it. If those applications are centrally controlled or manipulated, it won’t matter if the protocol is open. So it is up to the community to ensure that Bitcoin remains [decentralized](/guide/designing-products/principles/#decentralization) and allows for permissionless participation at the open-source application layer.
 
 ## Values
 
-In open design, we:
+Open-source software development has existed for decades. Public review and debate have helped it create better products and services. Open design shares similar values, though it is not as known or practiced.
+
+### Integrity
+
+- Make moral and responsible choices
+- Build with a principled foundation
+
+### Collaboration
+
+- Collaborate without competing
+- Approach problems with an open-mind
+- Share transparent work and open processes
```

Comment:
```suggestion
- Share your work and process in an open and transparent way
```
"transparent work" reads like the work is transparent. Did you mean to say the the sharing should be done in a transparent way?